docs/version0.30.1.md

```diff
@@ -0,0 +1,48 @@
+<!--
+* Copyright (c) 2022, 2022 IBM Corp. and others
```

Comment:
The first year should be 2017 in this repo: see https://github.com/eclipse-openj9/openj9-docs/blob/master/CONTRIBUTING.md.

Reply:
Updated.